scout/server/blueprints/dashboard/templates/dashboard/dashboard_general.html

```diff
@@ -1,100 +1,62 @@
+        <li class="nav-item"><a href="#" class="nav-link {% if panel == 'cases' %}active{% endif %}" data-toggle="tab" onclick="hide_div('cases')"><h4>Case statistics</h4></a></li>
+        <li class="nav-item"><a href="#" class="nav-link {% if panel == 'variants' %}active{% endif %}" data-toggle="tab" onclick="hide_div('variants')"><h4>Variant statistics</h4></a></li>
       </ul>
-    <div class="tab-content" id="tabs">
-      <div class="tab-pane" id="1">
-        {{ basic_stats() }}
-      </div>
-      <div class="tab-pane" id="2">
-        {{ cases_stats() }}
-      </div>
-      <div class="tab-pane" id="3">
-        {{ variants_stats() }}
-      </div>
-    </div>
-  </div>
 
-{% endblock %}
+      <!--filter panel -->
+      <div>
+        {{ dashboard_search_form() }}
+        {% if total_cases == 0 %}
+          <div class="alert alert-danger">Your search didn't return any results!</div>
+        {% endif %}
 
-{% macro basic_stats() %}
-<div class="card">
-  <div class="card-body">
-    {{ dashboard_search_form(1) }}
-    {{ general_stats_panels() }}
-  </div>
-</div>
-{% endmacro %}
+        {{ general_stats_panels() }}
+        {{ cases_stats_panels() }}
+        {{ variants_stats_panels() }}
+      </div>
 
-{% macro cases_stats() %}
-<div class="card">
-  <div class="card-body">
-    {{ dashboard_search_form(2) }}
-    {{ cases_stats_panels() }}
   </div>
-</div>
-{% endmacro %}
+{% endblock %}
+
 
-{% macro variants_stats() %}
+{% macro dashboard_search_form() %}
 <div class="card">
-  <div class="card-body">
-    {{ dashboard_search_form(3) }}<br><br>
-    {{ variants_stats_panels() }}
-  </div>
-</div>
-{% endmacro %}
+  <form class="mt-3 ml-3" id='form' class="form-horizontal" method='POST' action="{{ url_for('dashboard.index') }}" accept-charset="utf-8">
+      <input type="hidden" value="{{panel}}" name="panel">
+      <div class="form-row align-items-center mb-3">
+        <!--Search institute select-->
+        <div class="col-2 ml-3">
+          {{ dashboard_form.search_institute(class="form-control") }}
+        </div>
 
-{% macro dashboard_search_form(pane) %}
-<br>
-<form id='form' class="form-horizontal" method='POST' action="{{ url_for('dashboard.index') }}" accept-charset="utf-8">
-    <input type="hidden" value={{pane}} name="pane_id" id="pane_id">
-    <div class="row text-center">
-      <div class="col-md-4 col-sm-4">
-        {% if pane in (1,2) %}
-          <div class="input-group">
-            <span class="input-group-addon">
-              <span class="glyphicon glyphicon-search"></span>
-            </span>
-            <input type="search" class="form-control" value="{{ query if query }}" name="query" id="query" placeholder="Enter query to filter cases summarized below" onchange="update_select({% if current_user.is_admin %}true{% else %}false{% endif %})"></input>
+        {% if panel != "3" %}
```

Comment:
Oops? 😊 Sorry, my bad for making you change!

Reply:
Oh no, I must have missed that. I'll fix it in a new PR